src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -0,0 +1,173 @@
+        c.argument('topic_name', options_list=['--topic-name'], id_part='child_name_1', completer=get_topic_command_completion_list, help='Name of Topic')
+
+    for scope in ['servicebus topic subscription rule create', 'servicebus topic subscription rule update']:
+        with self.argument_context(scope) as c:
+            c.argument('action_sql_expression', arg_group='action', help='Action SQL expression.')
+            c.argument('action_compatibility_level', arg_group='action', type=int, help='This property is reserved for future use. An integer value showing the compatibility level, currently hard-coded to 20.')
+            c.argument('action_requires_preprocessing', options_list=['--enable-action-preprocessing'], arg_group='action', arg_type=get_three_state_flag(), help='A boolean value that indicates whether the rule action requires preprocessing.')
+            c.argument('filter_type', arg_type=get_enum_type(FilterType), help='Rule Filter types')
+            c.argument('filter_sql_expression', arg_group='sql_filter', help='SQL expression. e.g.')
+            c.argument('filter_requires_preprocessing', options_list=['--enable-sql-preprocessing'], arg_group='sql_filter', arg_type=get_three_state_flag(), help='A boolean value that indicates whether the rule action requires preprocessing.')
+            c.argument('action_compatibility_level', type=int, help='This property is reserved for future use. An integer value showing the compatibility level, currently hard-coded to 20.')
+            c.argument('correlation_id', arg_group='correlation_filter', help='Identifier of correlation.')
+            c.argument('message_id', arg_group='correlation_filter', help='Identifier of message.')
+            c.argument('to', arg_group='correlation_filter', help='Address to send to.')
+            c.argument('reply_to', arg_group='correlation_filter', help='Address of the queue to reply to.')
+            c.argument('label', arg_group='correlation_filter', help='Application specific label.')
+            c.argument('session_id', arg_group='correlation_filter', help='Session identifier')
+            c.argument('reply_to_session_id', arg_group='correlation_filter', help='Session identifier to reply to.')
+            c.argument('content_type', arg_group='correlation_filter', help='Content type of message.')
+            c.argument('requires_preprocessing', options_list=['--enable-correlation-preprocessing'], arg_group='correlation_filter', arg_type=get_three_state_flag(), help='A boolean value that indicates whether the rule action requires preprocessing.')
+
+    # Geo DR - Disaster Recovery Configs - Alias  : Region
+    with self.argument_context('servicebus georecovery-alias exists') as c:
+        c.argument('name', arg_type=name_type, help='Name of the Alias (Disaster Recovery) to check availability')
```

Comment:
Same as eventhub, recommend aliasing this to --alias/-a so it stays consistent.  Also, it currently doesn't appear under resource ID arguments, so I doubt the --ids scenario would work correctly.

Reply:
implemented